kmip/pie/objects.py

```diff
@@ -227,6 +235,7 @@ def __init__(self, algorithm, length, value, masks=None,
         self.value = value
```

Comment:
I agree with you on the type checking. Let's ignore that.

The checks in the future can use the cryptography library. We can load an object and if the bytes are not correct then cryptography will yell at us.